tests/resources/expected/test_if_else_swap/transformation_1.cpp

```diff
@@ -0,0 +1,37 @@
+int foo(int n) {
+    if (!(n == 1)) int k = 4;
+ else
+        {
+        int k = 0;
+        int j = 0;
+    }
+    if (!(n > 1)) {
+        int k = 5;
+        if (k < 5) {
+            k--;
+        }
+        return k;
+    } else {
+        int k = n + 5;
+        return 1;
+    }
+}
+
+int main() {
+    int x = foo(1);
+    if (x == 2) {
```

Comment:
As I understand, this if doesn't change. Is it complicated to swap multiple conditions?